src/Symfony/Component/ErrorHandler/ErrorHandler.php

```diff
@@ -270,13 +265,8 @@ public function setExceptionHandler(callable $handler = null)
 
     /**
      * Sets the PHP error levels that throw an exception when a PHP error occurs.
-     *
-     * @param int  $levels  A bit field of E_* constants for thrown errors
-     * @param bool $replace Replace or amend the previous value
```

Comment:
I would keep the information of both parameters, it is not obvious what kind of constant we should use here and what the replace argument does. Same for other methods.

Reply:
sure, re-added them 👍